.prettierrc.cjs

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  singleQuote: true,
+  trailingComma: 'all',
+  jsxSingleQuote: true,
+  printWidth: 120,
+};
+  
```

Comment:
O git tá acusando um erro aqui pois está faltando uma quebra de linha, a quebra de linha significa EOF (End Of File) e deve estar presente em todos os arquivos.